src/pykeen/utils.py

```diff
@@ -519,3 +519,25 @@ def format_relative_comparison(
 ) -> str:
     """Format a relative comparison."""
     return f"{part}/{total} ({part / total:2.2%})"
+
+
+BATCH_NORM_MODULES = (  # must be a tuple
+    torch.nn.BatchNorm1d,
+    torch.nn.BatchNorm2d,
+    torch.nn.BatchNorm3d,
+    torch.nn.SyncBatchNorm,
+)
+
+
+def get_batchnorm_modules(base: torch.nn.Module) -> Iterable[torch.nn.Module]:
```

Comment:
I love iterables too but this function will be much more usable if you make this return a list instead. The nice properties of chaining iterables and maps isn't so important for a short list like this one

Reply:
See 2ba1e362